lib/modules/datasource/cdnjs/index.ts

```diff
@@ -34,7 +34,10 @@ export class CdnjsDatasource extends Datasource {
 
   @cache({
     namespace: `datasource-${CdnjsDatasource.id}`,
-    key: ({ packageName }: GetReleasesConfig) => packageName.split('/')[0],
+    key: ({ packageName }: GetReleasesConfig) => {
+      const library = packageName.split('/')[0];
```

Comment:
do we really need this extra variable?

Reply:
It's not instantly obvious what does it mean, and looks ugly in the string template literal

Reply:
we should later extract it to a static function

Reply:
Static functions won't be available in the stage 3 decorators